index.html

```diff
@@ -0,0 +1,43 @@
+          alt=""
+          id="image"
+        />
```

Comment:
id="image"
        />
"The image closing tag should be immediately after the code, it should be  id="image"/>